File: calculator/calculator.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int num1;
    char op;
    int num2;

    // Welcome to smart Calculator
    printf("Welcome to the smart calculator \n");

    // Enter Number1
    num1 = get_int("Enter Number1: ");

    // Enter the Calculator Operator
    op = get_char("Enter the Calculator Operator?: ");

    // Enter Number2
    num2 = get_int("Enter Number2: ");

    // Calculator Operator
    if (op == '+')
    {
        printf("%i + %i = %i\n", num1, num2, num1 + num2);
    }
    else if (op == '-')
    {
        printf("%i - %i = %i\n", num1, num2, num1 - num2);
    }
    else if (op == '*')
    {
        printf("%i * %i = %i\n", num1, num2, num1 * num2);
    }
    else if (op == '/')
    {
        printf("%i / %i =%i\n", num1, num2, num1 / num2);
    }

}

File: calculator/calculator.py
#Enter Number1
Num1 = int(input("Enter Number1: "))
#Choose Operation type: * / + -
Operation = input("Choose Operation type: * / + - : ")
#Enter Number1
Num2 = int(input("Enter Number2: "))
def calculator(Operation):
    if Operation == "*":
        print(Num1, "*", Num2, "=", Num1 * Num2)
        return Num1 * Num2
    elif Operation == "/":
        if Num2 == 0:
            print("Error: Division by zero is not allowed.")
            return "Error: Division by zero is not allowed."
        else:
            print(Num1, "/", Num2, "=", Num1 / Num2)
            return Num1 / Num2
    elif Operation == "+":
        print(Num1, "+", Num2, "=", Num1 + Num2)
        return Num1 + Num2
    elif Operation == "-":
        print(Num1, "-", Num2, "=", Num1 - Num2)
        return Num1 - Num2
    else:
        return "Invalid operation"
Result = calculator(Operation)
